### ISSUE TITLE
Release ACE on Hugging Face

### ISSUE THREAD

**Issue by NielsRogge** [modelscope/scepter#49]
Hello @wenmengzhou and team 🤗 

I'm Niels and work as part of the open-source team at Hugging Face. I discovered ACE as it got featured on AK's daily papers: https://huggingface.co/papers/2410.00086. The paper page lets people discuss about your paper and lets them find artifacts about it (your model for instance) you can also claim the paper as yours which will show up on your public profile at HF.

Would you like to host the model you've pre-trained on https://huggingface.co/models? Hosting on Hugging Face will give you more visibility/better discoverability. We can add tags in the model cards so that people find the models easier, link it to the paper page, etc.

If you're down, leaving a guide [here](https://huggingface.co/docs/hub/models-uploading). If it's a PyTorch model, you can use [PyTorchModelHubMixin](https://huggingface.co/docs/huggingface_hub/package_reference/mixins#huggingface_hub.PyTorchModelHubMixin) class which adds from_pretrained and push_to_hub to the model which lets you to upload the model and people to download and use models right away.
If you do not want this and directly want to upload model through UI or however you want, people can also use [hf_hub_download](https://huggingface.co/docs/huggingface_hub/en/guides/download#download-a-single-file).

After uploaded, we can also link the models to the paper page (read [here](https://huggingface.co/docs/hub/en/model-cards#linking-a-paper)) so people can discover your model.

You can also build a demo to your model on [Spaces](https://huggingface.co/spaces) we can provide you an A100 grant.

What do you think?

Kind regards,

Niels

**Comment by hanzhn** [comment]
Hi Niels,

Thank you for reaching out and for your thoughtful offer.

We are excited to upload our models and create a Hugging Face Space! Running the ACE demo on an A100 would be fantastic! 🥳

Our plan is to release the ACE models by the end of October. We appreciate your patience as we complete our preparations.

Thank you once again for your support, and we look forward to collaborating with you!

Best regards,
Zhen Han，
Wanx Team, Alibaba Group

**Comment by LouieStark** [comment]
> Hello @wenmengzhou and team 🤗
> 
> I'm Niels and work as part of the open-source team at Hugging Face. I discovered ACE as it got featured on AK's daily papers: https://huggingface.co/papers/2410.00086. The paper page lets people discuss about your paper and lets them find artifacts about it (your model for instance) you can also claim the paper as yours which will show up on your public profile at HF.
> 
> Would you like to host the model you've pre-trained on https://huggingface.co/models? Hosting on Hugging Face will give you more visibility/better discoverability. We can add tags in the model cards so that people find the models easier, link it to the paper page, etc.
> 
> If you're down, leaving a guide [here](https://huggingface.co/docs/hub/models-uploading). If it's a PyTorch model, you can use [PyTorchModelHubMixin](https://huggingface.co/docs/huggingface_hub/package_reference/mixins#huggingface_hub.PyTorchModelHubMixin) class which adds from_pretrained and push_to_hub to the model which lets you to upload the model and people to download and use models right away. If you do not want this and directly want to upload model through UI or however you want, people can also use [hf_hub_download](https://huggingface.co/docs/huggingface_hub/en/guides/download#download-a-single-file).
> 
> After uploaded, we can also link the models to the paper page (read [here](https://huggingface.co/docs/hub/en/model-cards#linking-a-paper)) so people can discover your model.
> 
> You can also build a demo to your model on [Spaces](https://huggingface.co/spaces) we can provide you an A100 grant.
> 
> What do you think?
> 
> Kind regards,
> 
> Niels

Hi, we are ready to release our ACE model and intent to build a demo on huggingface. We would appreciate it if you could  provide us an A100 grant.

**Comment by LouieStark** [comment]
> Hello @wenmengzhou and team 🤗
> 
> I'm Niels and work as part of the open-source team at Hugging Face. I discovered ACE as it got featured on AK's daily papers: https://huggingface.co/papers/2410.00086. The paper page lets people discuss about your paper and lets them find artifacts about it (your model for instance) you can also claim the paper as yours which will show up on your public profile at HF.
> 
> Would you like to host the model you've pre-trained on https://huggingface.co/models? Hosting on Hugging Face will give you more visibility/better discoverability. We can add tags in the model cards so that people find the models easier, link it to the paper page, etc.
> 
> If you're down, leaving a guide [here](https://huggingface.co/docs/hub/models-uploading). If it's a PyTorch model, you can use [PyTorchModelHubMixin](https://huggingface.co/docs/huggingface_hub/package_reference/mixins#huggingface_hub.PyTorchModelHubMixin) class which adds from_pretrained and push_to_hub to the model which lets you to upload the model and people to download and use models right away. If you do not want this and directly want to upload model through UI or however you want, people can also use [hf_hub_download](https://huggingface.co/docs/huggingface_hub/en/guides/download#download-a-single-file).
> 
> After uploaded, we can also link the models to the paper page (read [here](https://huggingface.co/docs/hub/en/model-cards#linking-a-paper)) so people can discover your model.
> 
> You can also build a demo to your model on [Spaces](https://huggingface.co/spaces) we can provide you an A100 grant.
> 
> What do you think?
> 
> Kind regards,
> 
> Niels

I hope this message finds you well. We are excited to announce that we are ready to release our ACE model, and as part of our launch strategy, we are looking to build a demo on Hugging Face.

In order to facilitate this demonstration and showcase the capabilities of our model effectively, we would greatly appreciate your support in the form of an A100 grant. The A100 GPUs will enable us to harness the full potential of our model, ensuring a smooth and responsive user experience for our demo.

Thank you for considering our request. We are looking forward to the possibility of collaborating with you and contributing to the Hugging Face community.

Warm regards,

Wanx Team, Alibaba Group

**Comment by NielsRogge** [comment]
thanks for the update, will ping @hysts here. The best might be to leverage ZeroGPU (which is A100s which are shared among hundreds of Spaces, for free): https://huggingface.co/zero-gpu-explorers

**Comment by hysts** [comment]
Hi @LouieStark
Feel free to open a community grant ([docs](https://huggingface.co/docs/hub/en/spaces-gpus#community-gpu-grants)). We can assign ZeroGPU to your Space as a grant.

**Comment by LouieStark** [comment]
> Thank you for considering our request. We are looking forward to the possibility of collaborating with you and contributing to the Hugging Face community.

We are excited to share that we have developed a demo for ACE on Hugging Face Spaces and have submitted an application for the community grant. We would greatly appreciate your assistance in assigning ZeroGPU to our space. Thank you for your consideration.

**Comment by hysts** [comment]
@LouieStark Awesome! I just assigned ZeroGPU to the Space.

**Comment by hysts** [comment]
Hi @maochaojie Cool! I just assigned ZeroGPU!

**Comment by maochaojie** [comment]
Yes, we plan to release a new model of ACE(https://huggingface.co/scepter-studio/ACE-0.6B-1024px).  The "image-to-image" tag will make it more convenient for everyone to use our model and it's a great feature.

**Comment by JasonHuangChina** [comment]
> [@NielsRogge](https://github.com/NielsRogge) Yes, we plan to release a new model of ACE(https://huggingface.co/scepter-studio/ACE-0.6B-1024px). The "image-to-image" tag will make it more convenient for everyone to use our model and it's a great feature.

@maochaojie @hanzhn I’m really glad to be using an end-to-end model like ACE, but I’m not sure how to use Scepter for global fine-tuning. It seems to always start training from scratch. Do I need to change any parameters for this?
